src/Layout/Layout.component.tsx

```diff
@@ -0,0 +1,80 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+// Constants
+// ------------------------------------------------------------------------------------------------------------------
+export const DEFAULT_LAYOUT_WIDTH = '100%';
+export const DEFAULT_CONTENT_WIDTH = '71.25rem';
+export const TRANSPARENT = 'transparent';
+
+// Styled Components
+// ------------------------------------------------------------------------------------------------------------------
+const StyledLayout = styled.section<StyledLayoutProps>`
+    max-width: ${props => props.layoutWidth};
+    background-color: ${props => props.layoutBackgroundColor};
+    margin: 0 auto;
+`;
+
+const StyledContent = styled.div<StyledContentProps>`
+    max-width: ${props => props.contentWidth};
+    background-color: ${props => props.contentBackgroundColor};
+    margin: 0 auto;
+`;
+
+StyledContent.displayName = 'StyledContent';
+StyledLayout.displayName = 'StyledLayout';
+
+// Interfaces
+// ------------------------------------------------------------------------------------------------------------------
+interface LayoutProps {
+    /** The width of the layout, edge to edge. */
+    layoutWidth?: string;
+    /** The width of the content within the layout. */
+    contentWidth?: string;
+    /** The background color of the layout. Default is transparent. */
+    layoutBackgroundColor?: string;
+    /** The background color of the content within the layout area. Default is transparent. */
+    contentBackgroundColor?: string;
+    /** Additional classname. */
+    className?: string;
+    children?: any;
+}
+
+interface StyledLayoutProps {
+    layoutWidth?: string;
+    layoutBackgroundColor?: string;
+}
+
+interface StyledContentProps {
```

Comment:
Do we want to extricate `Content` from `Layout` as a nested component in a nested dir?

Reply:
The reason I have Layout and Content styled components is to more easily be able to change the width of both of them via the single Layout (or SidebarLayout) component(s). I don't think it's necessary (right now anyways) to make them individual components for the user to consume. Breaking them up like that removes some of the simplicity of just dropping in `Layout` or `SidebarLayout`. I'm open to discuss this more though, as perhaps I'm not seeing the same angle as you.

Reply:
👍